core/src/com/unciv/models/ruleset/Building.kt

```diff
@@ -765,6 +765,8 @@ class Building : RulesetStatsObject(), INonPerpetualConstruction {
             name -> true
             "Building", "Buildings" -> !isAnyWonder()
             "Wonder", "Wonders" -> isAnyWonder()
+            "National Wonder" -> isNationalWonder
+            "Non-national Wonder" -> isWonder
```

Comment:
I believe in Civ V these are called "World Wonders"— Though I think I lean towards "Non-national Wonder" myself.

Reply:
This is a bit confusing though, as it could be taken to mean "all buildings that are not national wonders".
"World wonder", "Global wonder", "International wonder"? "World" seems like the best fit